dockerrotate/main.py

```diff
@@ -62,9 +68,17 @@ def make_client(args):
     all the possible certificate options through argparse.
     """
     if args.use_env:
-        return Client(**kwargs_from_env(assert_hostname=False))
+        client = Client(version=args.client_version, **kwargs_from_env(assert_hostname=False))
     else:
-        return Client(base_url='unix://var/run/docker.sock')
+        client = Client(version=args.client_version, base_url='unix://var/run/docker.sock')
```

Comment:
Proposed changed worked into 5053ced
I'm usually 99% opposed to using star-args  because pylint discourages them; but pylint didn't seem to care for this repo so I assume the repo author accepts their risk already.

I also moved `unix_soc_args` to a constant.